src/services/task/routes.ts

```diff
@@ -28,19 +28,28 @@ export default [
       // req.body.payload
       // req.body.targets
       // req.body.target_regex
-      const responsePaaS = (await axios.post('http://localhost:5000/PaaS', {
-        ip_address: req.body.address,
+      req.body.addresses.forEach(async (address: string) => 
+      {
+        try{
+      const responsePaaS = (await axios.post('http://192.168.36.128:5000/PaaS', {
+        address: address,
         username: req.body.username ? req.body.username : "Administrator",
         steps: req.body.steps
       })).data;
-      const responseExecute = (await axios.post('http://localhost:5000/execute', {
-        ip_address: req.body.targets,
+      const responseExecute = (await axios.post('http://localhost:5001/execute', {
+        address: address,
         username: 'Witcher',
         password: 'Switcher',
         process: 'powershell.exe',
         command: req.body.payload,
       })).data;
       res.json(responseExecute);
```

Comment:
Shouldn't work, return res.json after the first address executed.
Need to save all the responses and return
{ <address>: <microservice response> }

Reply:
Added